<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventServiceCheckDialog</class>
 <widget class="QDialog" name="EventServiceCheckDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Контроль услуг и стандартов в обращениях</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="layoutFilter">
     <item>
      <widget class="QLabel" name="lblDateFrom">
       <property name="text">
        <string>с</string>
       </property>
       <property name="buddy">
        <cstring>edtDateFrom</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtDateFrom"/>
     </item>
     <item>
      <widget class="QLabel" name="lblDateTo">
       <property name="text">
        <string>по</string>
       </property>
       <property name="buddy">
        <cstring>edtDateTo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtDateTo"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblDate">
     <property name="text">
      <string>Дата назначения:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEventType">
     <property name="text">
      <string>Типы обращений:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="buddy">
      <cstring>tblEventType</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="CRBListBox" name="tblEventType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>4</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="CProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QListWidget" name="log">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>6</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="layoutButtons">
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Начать проверку</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>361</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Прервать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CProgressBar</class>
   <extends>QProgressBar</extends>
   <header>library/ProgressBar.h</header>
  </customwidget>
  <customwidget>
   <class>CRBListBox</class>
   <extends>QTableView</extends>
   <header>library/RBListBox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtDateFrom</tabstop>
  <tabstop>edtDateTo</tabstop>
  <tabstop>btnStart</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>log</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
